.eslint-doc-generatorrc.js

```diff
@@ -0,0 +1,11 @@
+/** @type {import('eslint-doc-generator').GenerateOptions} */
```

Comment:
let's add this file to publishConfig.ignore in package.json?

Reply:
Done.